init.lua

```diff
@@ -3,9 +3,10 @@
 -- Copyright © 2014-2020 4aiman, Hugo Locurcio and contributors - MIT License
 -- See `LICENSE.md` included in the source distribution for details.
 
-if minetest.settings:get_bool("health_bars") == false or
-		not minetest.settings:get_bool("enable_damage")
-then return end
+local enabled = minetest.settings:get_bool("health_bars")
```

Comment:
Not sure if this is the `true` by default. Just a suggestion.

Reply:
To get `true` by default, use `get_bool("xxx") ~= false`